### PR TITLE
Initial updates

### PR DESCRIPTION
**Pull request by asciifaceman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
-Jave Language Basic Constructs:
-Assignment:
+Jave is a language predicated on the idea of human readable code as conversation.
+
+### Jave Language Basic Constructs
+
+#### Assignment:
 ```
-x :-_- 5
+x is identical to 5
```
Comment (smaslennikov) [inline README.md]: But I kind of like the -_- operator

**Review by slavaaaaaaaaaa — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (asciifaceman) [reply README.md]: I thought about it, but I want to avoid symbols as they are not conversational

Comment (asciifaceman) [reply README.md]: I was going to go out of left field with like

x ∆ƒåå 5 type shit

File: README.md [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
 ```
-Conditional
+
+#### Conditional
 ```
 is x identical to 1 ?
   y is identical to 3
+or is x identical to 2 ?
+  y is identical to 1
```
Comment (smaslennikov) [inline README.md]: maybe this should be equal rather than identical

but who cares

Comment (smaslennikov) [reply README.md]: assignment vs comparison

Comment (smaslennikov) [reply README.md]: perfect score

File: README.md [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
 x is identical to the value of x multiplied by the identical value of 5
-```
-```
 x is identical to the value of x after subtracting the value of 5
 ```
+
+Advanced while loop with array assignment
 ```
 y is identical to an array of values consisting of 0 and 0 and 0 and 0 and 0 and 0 and 0 and 0 and 0 and 0
 during x being less than identical to the value of 10 engage
```
Comment (smaslennikov) [inline README.md]: omg yes

**Comment by smaslennikov** [comment]
how do i mega approve this

File: README.md [hunk 4]
```diff
@@ -48,6 +59,28 @@ during x being less than identical to the value of 10 engage
 fin
 ```
 
+#### Objects and Arrays
+```
+Albus.Dumbledore is identical to a group of intrinsically related values consisting of 
+    Sonny.Moore which contains an array values consisting of 0 and 0 and 1
```
Comment (smaslennikov) [inline README.md]: *of values

**Review by slavaaaaaaaaaa — CHANGES_REQUESTED** [review]

**Review by slavaaaaaaaaaa — APPROVED** [review]

**Comment by asciifaceman** [comment]
I can't merge btw your perms are fecked

**Comment by slavaaaaaaaaaa** [comment]
Your privs should be fixed now though.

**Pull request merged**